test/scenarios/create_dir_with_utf8_character/scenario.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+
+/*:: import type { Scenario } from '..' */
+
+//const save = 'Partages reçus/'
+
+module.exports = ({
+  actions: [{ type: 'mkdir', path: 'Partages reçus' }],
+  expected: {
+    tree: ['Partages reçus/'],
```

Comment:
Encoded with NFC but local expectations will be normalized using NFD when tests are run on a HFS+ volume (see https://github.com/cozy-labs/cozy-desktop/pull/1684/files#diff-e535224c0b54370fef0dc6d963d6c27aR304)